src/index.html

```diff
@@ -132,7 +147,38 @@ <h2 class="modal__title">
       </div>
     </div>
 
-    <script defer src="./scripts/index.js"></script>
-    <script defer src="./scripts/validation.js"></script>
+    <div class="modal" id="edit-avatar-modal">
+      <div class="modal__content">
+        <button type="button" class="modal__close modal__close-button"></button>
+        <h2 class="modal__title">
+          Edit Profile Avatar
+        </h2>
+        <form id="edit-avatar-form" class="modal__form" novalidate>
+          <label for="profile-avatar-input" class="modal__label">
+            Profile<picture></picture>
+            <input
```

Comment:
* Please, add `type="url"` for the `avatar` input, to validate it as a `link` https://snipboard.io/YQ0XiB.jpg 


```html
            <input
              type="url"

```